src/sentry/notifications/activity/release.py

```diff
@@ -116,12 +119,18 @@ def get_category(self) -> str:
     def fine_tuning_key(self) -> str:
         return "deploy/"
 
-    def get_message_description(self) -> str:
-        text = ""
+    def get_message_description(self) -> List[Dict[str, Any]]:
+        buttons = []
         if self.release:
             for project in self.projects:
                 project_url = absolute_uri(
                     f"/organizations/{self.organization.slug}/releases/{self.version}/?project={project.id}&unselectedSeries=Healthy/"
                 )
-                text += f"* <{project_url}|{project.slug}>\n"
-        return text.rstrip()
+                buttons.append(
```

Comment:
This is Slack specific logic can we add a new function like `get_projects()` and turn them into buttons elsewhere?